app/src/main/java/ch/epfl/sdp/blindly/dependency_injection/UserRepositoryModule.kt

```diff
@@ -16,15 +23,33 @@ import javax.inject.Singleton
 @InstallIn(SingletonComponent::class)
 object UserRepositoryModule {
 
+    val firestore = Firebase.firestore

```

Comment:
cirrus tell is that there is a memory leek here, maybe we should take that into consideration